### PR TITLE
Support for programmatically adding mocks

### PR DESCRIPTION
**Pull request by sqren**
#It's now possible to run mock server programmatically like:

```js
const {mockServer} = require('http-mockserver');

mockServer
    .addMocks([...])
    .start();
```

This change is backwards compatible, so doesn't require any changes of clients.

@Tradeshift/collaboration

### CODE REVIEW

File: bin/configInitializer.js [hunk 1]
```diff
@@ -1,57 +0,0 @@
-const fs = require('fs');
```
Comment (sqren) [inline bin/configInitializer.js]: This file was renamed `mockFileReader.js`, and then some minor refactoring were done.

File: src/server/index.js [hunk 2]
```diff
@@ -1,27 +1,15 @@
...
-
-app.get('/listener/:port?', controller.getListeners);
-app.delete('/clear', controller.clear);
-
-app.get('/requests/:port?', controller.getRequestLogs);
-
-app.use((err, req, res, next) => {
-	console.error(err.stack);
-	res.status(400).send(err.message);
-});
```
Comment (sqren) [inline src/server/index.js]: Moved all the route declarations into `expressServer.js`

**Comment by andrerigon** [comment]
👂

**Comment by gogozby** [comment]
🍼

**Pull request merged**